dist/definitions.json

```diff
@@ -326,6 +326,17 @@
     "pattern": "^(\\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)$",
     "type": "string"
   },
+  "dischargeType": {
+    "type": "string",
+    "enum": [
```

Comment:
These might end up matching the codes that Preneed uses.

Reply:
I'm using this in the VR&E schema as well. I'll migrate those to use this once this is merged in.